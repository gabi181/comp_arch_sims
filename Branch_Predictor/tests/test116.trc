8 8 8 1 global_history global_tables using_share_lsb
0x3E62B4FC T 0x45A8C534
0x0387614C N 0x51A40700
0x3E62B4FC N 0x45A8C534
0x0387614C T 0x51A40700
0x3E62B4FC T 0x45A8C534
0x3E62B4FC N 0x45A8C534
0x3E62B4FC N 0x45A8C534
0x3E62B4FC T 0x45A8C534
0x3E62B4FC T 0x45A8C534
0x3E62B4FC T 0x45A8C534
0x0387614C T 0x51A40700
0x3E62B4FC N 0x45A8C534
0x0387614C N 0x51A40700
0x3E62B4FC N 0x45A8C534
0x3E62B4FC T 0x45A8C534
0x3E62B4FC T 0x45A8C534
0x3E62B4FC T 0x45A8C534
0x0387614C T 0x51A40700
0x3E62B4FC T 0x45A8C534
0x3E62B4FC N 0x45A8C534
0x3E62B4FC N 0x45A8C534
0x3E62B4FC N 0x45A8C534
