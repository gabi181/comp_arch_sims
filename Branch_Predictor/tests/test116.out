0x3e62b4fc N 0x3e62b500
0x387614c N 0x3876150
0x3e62b4fc N 0x3e62b500
0x387614c N 0x3876150
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x387614c N 0x3876150
0x3e62b4fc N 0x3e62b500
0x387614c N 0x3876150
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x387614c N 0x3876150
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
0x3e62b4fc N 0x3e62b500
flush_num: 12, br_num: 22, size: 832b
